docs/api.md

```diff
@@ -33,6 +33,8 @@ Documentation for other releases can be found at
 
 # The Kubernetes API
 
+For existing and future workloads, we want to provide a consistent, stable set of APIs, over which developers can build and extend Kubernetes. This includes input validation, a consistent API structure, clean semantics, and improved diagnosability of the system. 
```

Comment:
In the interest of keeping our documentation lightweight, I would suggest removing this. Maybe we can have a general philosophy doc somewhere (we talked about doing that at one point, I think).